### PR TITLE
Add pytorch1.3 deprecated warning in README

### PR DESCRIPTION
**Pull request by ly015**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ MMPose is an open-source toolbox for pose estimation based on PyTorch.
 It is a part of the [OpenMMLab project](https://github.com/open-mmlab).
 
 The master branch works with **PyTorch 1.3+**.
+Please kindly note that MMPose will switch to **PyTorch 1.5+** from Oct. 2021. The compatibility to earlier versions of PyTorch will no longer be guaranteed.
```
Comment (innerlee) [inline README.md:21]: Modify L20 to the current value

Comment (ly015) [reply README.md:21]: What does the current value mean?

Comment (innerlee) [reply README.md:21]: image

Comment (innerlee) [reply README.md:21]: Okay the modification could be delayed to October

**Review by innerlee — APPROVED** [review]

**Pull request merged**